package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@gofynd/fp-signature",
+  "version": "0.0.1",
+  "description": "Fynd platform's signature logic",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/mjs/index.js",
+  "browser": "./dist/web/fp-signature.min.js",
+  "types": "./dist/types/index.d.ts",
+  "scripts": {
+    "build": "npm run clear && concurrently 'npm:build:cjs' 'npm:build:mjs' 'npm:build:types' 'npm:build:web'",
+    "build:cjs": "tsc -p tsconfig-cjs.json",
+    "build:mjs": "tsc -p tsconfig-mjs.json",
+    "build:web": "webpack",
+    "build:types": "tsc -p tsconfig-types.json",
+    "prepublishOnly": "npm run build",
+    "clear": "rimraf ./dist",
+    "test": "jest --coverage"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gofynd/fp-signature.git"
+  },
+  "homepage": "https://github.com/gofynd/fp-signature.git",
+  "author": {
+    "name": "Fynd Developer",
+    "email" : "dev@fynd.com"
+  },
+  "license": "ISC",
+  "dependencies": {
+    "crypto-js": "^4.1.1",
+    "querystring-es3": "^0.2.1"
```

Comment:
check if this can be moved into dev dependencies @bhargavprajapatiFynd

Reply:
Moved querystring-es3 to devdependency